lib/beaker/cli.rb

```diff
@@ -69,11 +69,15 @@ def print_version_and_options
       @logger.info(@options.dump)
     end
 
+    def create_network_manager
```

Comment:
Yeah if I recall correctly, provision on the network manager requests new VMs, which means if I was to do that in destroy, we'd have the VMs obtained in `beaker provision`, then we'd request new VMs as part of `beaker destroy`, then destroy them right away, but the VMs obtained in `beaker provision` would still be allocated. So instead I just create a new network manager instance, then update the state to match whatever was configured during `beaker provision`, and call destroy so that the VMs obtained during `beaker provision` are destroyed